test/fail_compilation/checkimports3.d

```diff
@@ -0,0 +1,15 @@
+/*
+REQUIRED_ARGS: -transition=checkimports -de
+TEST_OUTPUT:
+---
+fail_compilation/checkimports3.d(14): Deprecation: local import search method found overloadset checkimports3.foo (3 overloads) instead of overloadset checkimports3.foo (2 overloads)
```

Comment:
I don't understand how this helps diagnose the problem.
BTW, that does `local import search method` mean? There's no local imports involved what I can see.